pkg/network/additional_networks.go

```diff
@@ -0,0 +1,58 @@
+package network
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"github.com/pkg/errors"
+	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	netv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+)
+
+// renderMultus returns the manifests of Multus.
+func renderMultus(manifestDir string) ([]*uns.Unstructured, error) {
```

Comment:
After looking into the change, I remembered that the reason I separated the rendering of Multus image/daemonset/network attachment crd and ``RawCNIConfig`` into two functions is that there might be more than one additional network which require multiple rendering of the single RawCNIConfig [yaml format](https://github.com/openshift/cluster-network-operator/pull/22/commits/3b1739821983e0536d735a491879787e1b79b9fa#diff-cbae04e411825079bca3a2de8dd75b12),  but Multus image/daemonset/crd manifests will only need to be applied once, so I had two functions to render manifests from two different folders, one needs to be recursive rendering, the other is just applied once.

Another reason we are considering is about enabling openshift SDN migration with Multus in the future. Meaning we might want to have Multus deployed even without additional networks (but Multus will still apply to the NetworkAttachmentDefinition standard, it's just that it can be deployed without providing additional network configs; sorry for the confusion made from last comment), and migrate the default CNI config of Multus from one SDN solution to another, for example, from Calicao to OVN. so having one ``renderMultus`` function allows us to easily transfer to support the migration.

With above, I think perhaps we could change the name of ``renderMultus`` to sth else, like ``renderMultusCNI``, ``renderMultusDasmonSet`` or ``renderMultusCNIAndCRD`` etc, thoughts?

Reply:
Updated a new version of PR, changing ``renderMultus`` to ``renderMultusConfig`` as I think it's all about rendering necessary pieces of config for Multus to start function. also renamed the bindata sub-folder names to:

Multus configs: bindata/network/additional-networks/multus/
Additional custom resource configs of network attachment definition kind: bindata/network/additional-networks/cr/